### PR TITLE
[MRG] Spelling fixes

### PR DESCRIPTION
**Pull request by betatim**
Not sure about "loszulegen" vs "los zulegen" but my spell checker suggested the new version

### CODE REVIEW

**Comment by fursinger** [comment]
Duden says loszulegen in 1 word.
https://www.duden.de/rechtschreibung/loslegen

File: pages/index.vue [hunk 1]
```diff
@@ -117,7 +117,7 @@
                   <h3 class="headline">Mit Kreditkarte bezahlen</h3>
                   <p>
                     Sie brauchen keine Rechnung abzuwarten, um mit dem Signieren
-                    loszulegen.
+                    los zulegen.
```
Comment (betatim) [inline pages/index.vue:120]:
```suggestion
                    loszulegen.
```
Duden says Firefox spellchecker needs to go back to school :)

File: pages/index.vue [hunk 2]
```diff
@@ -31,7 +31,7 @@
               </p>
               <p>
                 Um Mit QES signieren zu können, muss man seine Identität
```
Comment (fursinger) [inline pages/index.vue:33]: `mit` should be kleingeschrieben.

Comment (betatim) [inline pages/index.vue:33]:
```suggestion
                Um mt QES signieren zu können, muss man seine Identität
```

**Pull request merged**